src/components/dictionaryConcepts/components/ConceptDropdown.jsx

```diff
@@ -15,7 +15,7 @@ const ConceptDropdown = props => (
         Add existing concepts
       </button>
       <div className="dropdown-menu" aria-labelledby="dropdownMenuButton">
-        <a className="dropdown-item" href="/dashboard/concepts/hadijah315/ICD-10">
+        <a className="dropdown-item" href="/dashboard/concepts/Organization/CIEL/CIEL">
```

Comment:
Kindly use react-router's `Link` to avoid full page re-render.

Reply:
@emasys I suggest changing it to link needs a different PR because it will affect the tests,  I only had to change the url to a working one.

Reply:
I hope we remember to fix this and it doesn't become a `broken window`.

Reply:
We shall fix it but thanks for pointing that out